libkbfs/keybase_daemon_local.go

```diff
@@ -535,7 +535,7 @@ func (k *KeybaseDaemonLocal) addTeamKeyForTest(
 func (k *KeybaseDaemonLocal) addTeamsForTestLocked(teams []TeamInfo) {
 	for _, t := range teams {
 		k.localTeams[t.TID] = t
-		k.asserts[string(t.Name)] = t.TID.AsUserOrTeam()
+		k.asserts[string(t.Name)+"@team"] = t.TID.AsUserOrTeam()
```

Comment:
The issue is [here](https://github.com/keybase/kbfs/blob/a768f81ba72da33f0d82d5c20d407337e540063e/libkbfs/keybase_daemon_local.go#L180).  When checking any kind of key/value assertion, this code changes it into the '@' format.  This local daemon doesn't really support "key:value" style assertions, but it seems more natural to do that for teams.